app/uk/gov/hmrc/timetopay/arrangement/config/microserviceWiring.scala

```diff
@@ -43,54 +39,51 @@ object MicroserviceAuthConnector extends AuthConnector with ServicesConfig {
   override val authBaseUrl = baseUrl("auth")
 }
 
-object DesArrangementApiService extends DesArrangementService with ServicesConfig {
+class DesArrangementApiService() extends DesArrangementService with ServicesConfig {
   override val desArrangementUrl: String = baseUrl("des-arrangement-api")
   override val serviceEnvironment: String = getConfString("des-arrangement-api.environment", "unknown")
   override val authorisationToken: String = getConfString("des-arrangement-api.authorization-token", "not-found")
 
   override val http: HttpGet with HttpPost = WSHttp
 }
 
-object DesTTPArrangementBuilder extends DesTTPArrangementBuilder {}
-
-object RepositoryConfig  {
-  private implicit val connection = {
-    import play.api.Play.current
-    ReactiveMongoPlugin.mongoConnector.db
+class LetterAndControlAndJurisdictionCHecker extends ServicesConfig {
+  def createLetterAndControlConfig:LetterAndControlConfig = {
+    LetterAndControlConfig(
+      getConfString("letterAndControl.salutation", "sfsds"),
```

Comment:
We need to think of more appropriate second messages, 'sfsds' accidentally got left in